gtn/experimental/CMakeLists.txt

```diff
@@ -1,6 +1,4 @@
-cmake_minimum_required(VERSION 3.18)
-
-
+cmake_minimum_required(VERSION 3.17)
```

Comment:
just curious, is there any reason to keep this version different from `gtn/CMakeLists.txt`